tests/tests.js

```diff
@@ -711,32 +711,42 @@ QUnit.test("validateForm test", function(assert) {
 });
 
 QUnit.module( "CheckIt tests for functions that involve store", {
-  before: function() {
-    // prepare something once for all tests
-  },
   beforeEach: function() {
     // prepare something before each test
+    this.$fixture = $('#qunit-fixture');
+    this.$calendarDiv = this.$fixture.find('#calendarDiv');
+    
+    this.$fixture.append(
+        `<script>
+        $(document).ready( function() {
+        checkit = new CheckIt('localStorage');
```

Comment:
Well, it must be the same because I don't initialize checkit anywhere else but in the script tag. I just wanted it to be as close to as it would work on the actual page as possible.